doc/user/content/self-managed-deployments/installation/install-on-azure.md

```diff
@@ -0,0 +1,314 @@
+- AKS clusters
+- Azure PostgreSQL Flexible Server instances
+- Storage accounts
+- Virtual networks and networking resources
+- Managed identities and role assignments
+
+### Required Tools
+
+- [Terraform](https://developer.hashicorp.com/terraform/install?product_intent=terraform)
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Helm 3.2.0+](https://helm.sh/docs/intro/install/)
+
+### License Key
+
+{{< yaml-table data="self_managed/license_key" >}}
+
+## Getting started: Simple example
+
+{{< warning >}}
+
+{{< self-managed/terraform-disclaimer >}}
+
+{{< /warning >}}
+
+### Step 1: Set Up the Environment
+
+1. Open a terminal window.
+
+1. Clone the Materialize Terraform repository and go to the
+   `azure/examples/simple` directory.
+
+   ```bash
+   git clone https://github.com/MaterializeInc/materialize-terraform-self-managed.git
+   cd materialize-terraform-self-managed/azure/examples/simple
+   ```
+
+1. Authenticate with Azure.
+
+    ```bash
+    az login
+    ```
+
+   The command opens a browser window to sign in to Azure. Sign in.
+
+1. Select the subscription and tenant to use. After you have signed in, back in
+   the terminal, your tenant and subscription information is displayed.
+
+    ```none
+    Retrieving tenants and subscriptions for the selection...
+
+    [Tenant and subscription selection]
+
+    No     Subscription name    Subscription ID                       Tenant
+    -----  -------------------  ------------------------------------  ----------------
+    [1]*   ...                  ...                                   ...
+
+   The default is marked with an *; the default tenant is '<Tenant>' and
+   subscription is '<Subscription Name>' (<Subscription ID>).
+   ```
+
+   Select the subscription and tenant.
+
+### Step 2: Configure Terraform Variables
+
+1. Create a `terraform.tfvars` file with the following variables:
+
+   - `subscription_id`: Azure subscription ID
+   - `resource_group_name`: Name for the resource group to create (e.g.
+     `mz-demo-rg`)
+   - `name_prefix`: Prefix for all resource names (e.g., `simple-demo`)
+   - `location`: Azure region for deployment (e.g., `westus2`)
+   - `license_key`: Materialize license key
+   - `tags`: Map of tags to apply to resources
+
+   ```hcl
+   subscription_id     = "your-subscription-id"
+   resource_group_name = "mz-demo-rg"
+   name_prefix         = "simple-demo"
+   location            = "westus2"
+   license_key         = "your-materialize-license-key"
+   tags = {
+     environment = "demo"
+   }
+   # internal_load_balancer = false   # default = true (internal load balancer). You can set to false = public load balancer.
+   # ingress_cidr_blocks = ["x.x.x.x/n", ...]
+   # k8s_apiserver_authorized_networks  = ["x.x.x.x/n", ...]
+   ```
+
+   {{% include-from-yaml data="self_managed/installation"
```

Comment:
Almost ... awaiting on resolution of https://github.com/MaterializeInc/materialize-terraform-self-managed/issues/129